### ISSUE TITLE
Scratchr2: My Stuff Show incorrect date

### ISSUE THREAD

**Issue by GrahamSH-LLK** [LLK/scratch-www#4205]
I have edited my excuse generator project recently. Still, it says 
Excuse Generator —— Version 2.2
Last modified: 11 Aug 2019
![image](https://user-images.githubusercontent.com/64214252/87588059-08f4e680-c6b1-11ea-9118-21bec2805ca3.png)
I tried editing it, and it didn't change. Finally, it changed to today.

### Steps to Reproduce
Go to my stuff
See if any incorrect dates are shown

Excuse Generator —— Version 2.2
Last modified: 11 Aug 2019

### Operating System and Browser

ChromeOS

**Comment by BryceLTaylor** [comment]
I suspect that what you're seeing is caching behavior.

**Comment by BryceLTaylor** [comment]
Hmm... it updated immediately for me. Is it just certain times that it doesn't update immediately for you?

**Comment by R4356th** [comment]
> 
> 
> I suspect that what you're seeing is caching behavior.

I agree as I cannot reproduce this after clearing my cache.

**Comment by Kenny2github** [comment]
I'm getting a more extreme version of this:
![image](https://user-images.githubusercontent.com/28599280/87746252-acb1d400-c822-11ea-961a-ba6b8fa9d91e.png)
It's worth noting that "Q&A!" was actually recently unshared, which is why it was recently modified - and yet it still displays 2017.

Also worth noting is that My Stuff is in "last modified" order, so the projects are being sorted correctly, but the displayed dates are incorrect.

TL,DR can reproduce

**Comment by GrahamSH-LLK** [comment]
It can't be cached, because I didn't have this computer in 2019!

**Comment by Explosion-Scratch** [comment]
Shouldn't this go in the bugs and glitches forum on scratch since this is a Scratchr2 issue?

**Comment by GrahamSH-LLK** [comment]
True, but I feel like GitHub is a much better way to report them. I want to keep it here.

**Comment by apple502j** [comment]
@GrahamSH-LLK It's not - you should post them on Bugs and Glitches
